checker/raw_result.go

```diff
@@ -34,6 +35,52 @@ type RawResults struct {
 	LicenseResults              LicenseData
 }
 
+// TODO: Add Msg to all results.
+
+// OutcomeType represents the type of the raw result.
+type OutcomeType int
```

Comment:
IMO, severity should get tagged on during evaluation phase. Severity is usually associated with a policy failure and raw results should remain agnostic to any policy. Wdut?

Reply:
changed made. PTAL